doc/changelog.rst

```diff
@@ -51,6 +51,9 @@ PyMongo 4.13 brings a number of changes including:
   or the `migration guide <https://www.mongodb.com/docs/languages/python/pymongo-driver/current/reference/migration/>`_ for more information.
 - Fixed a bug where :class:`pymongo.write_concern.WriteConcern` repr was not eval-able
   when using ``w="majority"``.
+- Ignored bits in a BSON BinaryVector of PACKED_BIT dtype should be set to zero.
```

Comment:
"Ignored bits" refers to the padding bits right? Should we say "padding" to avoid introducing a new term?

Reply:
Will update to clarify.